docs/reference/indices/index-mgmt.asciidoc

```diff
@@ -80,8 +82,10 @@ them.
 To view more information about a data stream, such as its generation or its
 current index lifecycle policy, click the stream's name.
 
+To view and explore the data within a data stream, click the compass symbol next to the data stream name to open {kibana-ref}/discover.html[Discover].
+
 [role="screenshot"]
-image::images/index-mgmt/management_index_data_stream_stats.png[Data stream details]
+image::images/index-mgmt/management_index_data_stream_stats2.png[Data stream details]
```

Comment:
Is there a reason we didn't just replace the existing image? I'd at least remove the old `management_index_data_stream_stats.png` image.